Sources/Int+Repetition.swift

```diff
@@ -0,0 +1,54 @@
+public extension Int {
+    
+    /// Runs a closure the callee's number of times.
+    ///
+    /// - Parameter closure: The closure to execute
+    public func times(_ closure: () -> Void) {
```

Comment:
only from the comment above i can tell what the function does. maybe something like `repeat` would make more sense ?

Reply:
i see now that the title is copied from old version. i have never used it before, so was confused, but my opinion stays same :)

Reply:
Lemme think on it - I do feel like `repeat` makes more sense if the number of times it's going to be repeated - ex `repeat(5) { ... }`, but there might be a better wording for this 🤔

Reply:
Yeah I think I wanna stick with `times` for now - I think `5.times` reads better than `5.repeat` or `5.repetitions`